textfield/styles/index.styl

```diff
@@ -19,12 +23,9 @@
   :invalid
     box-shadow                              none
 
-  textarea
-    vertical-align                          top
-
-  &.disabled
+  &--disabled
     .mdl-textfield__label:after
       display                               none
     
     .mdl-textfield__input
-      color                                 mdl-light-full
+      color                                 rgba(0,0,0,0.26)
```

Comment:
Hmmm... What's your reasoning behind this? Hard coded colors seems off, but then I don't know the full picture ofc.